### PR TITLE
[feature] Adding option to install wrapper for non-privileged user on systemd ...

### LINKED ISSUE #1138: Fix wrapper issues
* Settings chosen on install time, eg max_memory, is not reflected in tools/yajsw/conf/wrapper.conf making the db be run with default memory allocation, currently 1024 MB by the wrapper.
* No dynamic wrapper.conf settings at all from template wrapper.conf.install were expanded on unix platforms.
* The wrapper is always running as a privileged user eventhough eXist-db can be run as non-privileged user if specified in tools/yajsw/conf/wrapper.conf.
* Documentation is still referring to the old service wrapper so clients have problem installing the wrapper at all.

### PR DESCRIPTION
**Pull request by ljo**
… systems in tools/yajsw/bin/installDaemon.sh. Please note if you are running straight from repo, rather than using the installer, you need to manually expand the tools/yajsw/conf/wrapper.conf.install template to tools/yajsw/conf/wrapper.conf (including variables).
Also fixes #1138
See also updated documentation https://github.com/eXist-db/documentation/pull/78

### CODE REVIEW

File: tools/yajsw/bin/installDaemon.sh [hunk 1]
```diff
@@ -30,5 +30,53 @@ if [ ! -x "$PRGDIR"/"$EXECUTABLE" ]; then
   exit 1
 fi
 
-sudo "$PRGDIR"/"$EXECUTABLE"
+# systemd stuff 20160901/ljo
+function use_systemd {
+    echo "Using systemd with template non-privileged";
+    echo -e "\nPlease notice the environment variables JAVA_HOME, EXIST_HOME, and USER are used, so set these variables and rerun if they are not set to what you want.\n\n";
```
Comment (joewiz) [inline tools/yajsw/bin/installDaemon.sh]: Suggested change: "notice" > "note that"

**Comment by joewiz** [comment]
Looks like a really valuable PR. Thanks @ljo!

File: tools/yajsw/bin/installDaemon.sh (hunk 1 above)
Comment (joewiz) [inline tools/yajsw/bin/installDaemon.sh]: Great. No other changes from me!

**Comment by adamretter** [comment]
@ljo Is this considered "ready" now?

**Comment by ljo** [comment]
@adamretter Yes, ready.

**Pull request merged**